cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -259,6 +263,11 @@ public static EntityTopicOperator fromCrd(Kafka kafkaAssembly) {
         varList.add(buildEnvVar(ENV_VAR_TOPIC_METADATA_MAX_ATTEMPTS, String.valueOf(topicMetadataMaxAttempts)));
         varList.add(buildEnvVar(ENV_VAR_TLS_ENABLED, Boolean.toString(true)));
         varList.add(buildEnvVar(ENV_VAR_STRIMZI_GC_LOG_ENABLED, String.valueOf(gcLoggingEnabled)));
+        if (javaSystemProperties != null) {
+            varList.add(buildEnvVar(ENV_VAR_STRIMZI_JAVA_SYSTEM_PROPERTIES, ModelUtils.getJavaSystemPropertiesToString(javaSystemProperties)));
+        }
+        heapOptions(varList, 0.5, 5L * 1024L * 1024L * 1024L);
```

Comment:
So if i remember correctly, this tries to calculate the defult -Xmx etc.? I do not hink we want that, I think we want to just pass the options over to the containers / JVM.